ynput/OpenPype
# Fusion: Saver creator settings

## Changelog Description
Adding Saver creator settings and enhanced rendering path with template.

## Testing notes:
1. Open Fusion at test task 
2. Create Saver family
3. all should work as expected
